src/frontend/src/pages/PIAListPage/index.tsx

```diff
@@ -1,12 +1,30 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faPlus, faMagnifyingGlass } from '@fortawesome/free-solid-svg-icons';
 import EmptyPIAList from '../../components/public/EmptyPIAList';
 import PIAListTable from '../../components/public/PIAListTable';
 import { usePIALookup } from '../../hooks/usePIALookup';
+import InputText from '../../components/common/InputText/InputText';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 const PIAList = () => {
+  const history = useNavigate();
   const tableHeadings = ['Title', 'Last modified', 'Drafter', 'PIA status'];
   const { tableData } = usePIALookup();
+
+  const [searchText, setSearchText] = useState('Search by title or drafter');
+  const handleClearSearchText = () => {
+    setSearchText('');
+    history('');
```

Comment:
history('') - what does this mean?

Reply:
bad naming, let's me change to navigate instead